src/biokbase/narrative/tests/test_jobmanager.py

```diff
@@ -390,23 +415,159 @@ def test_cancel_jobs(self):
             self.jm.cancel_job(JOB_COMPLETED)
 
     @mock.patch("biokbase.narrative.jobs.jobmanager.clients.get", get_mock_client)
-    def test_retry_job_good(self):
-        retry_results = self.jm.retry_jobs([JOB_TERMINATED])
+    def test_retry_jobs__success(self):
```

Comment:
These retry_jobs tests can be refactored as they sort of follow a similar pattern